src/components/FlowRunResumeModalV2.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <p-modal v-if="flowRun" v-model:showModal="showModal" title="Resume Flow Run">
+    <p-message v-if="serverValidationError" error>
+      {{ serverValidationError }}
+    </p-message>
+
+    <template v-if="inputSchema">
+      <p-markdown-renderer v-if="inputDescription" :text="inputDescription" />
+    </template>
+
+    <p v-else>
+      Do you want to resume this flow run?
+    </p>
+
+    <template v-if="inputSchema">
+      <SchemaInputV2 v-model:values="parameters" :schema="inputSchema" :errors="[]" :kinds="['none', 'json', 'workspace_variable']" />
+    </template>
+
+    <template #actions>
+      <p-button primary @click="resume">
+        Submit
+      </p-button>
+    </template>
+  </p-modal>
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { useValidationObserver } from '@prefecthq/vue-compositions'
+  import { ref } from 'vue'
+  import { useFlowRun, useWorkspaceApi } from '@/compositions'
+  import { localization } from '@/localization'
+  import { SchemaV2, SchemaValuesV2, SchemaInputV2 } from '@/schemas'
+  import { getApiErrorMessage } from '@/utilities/errors'
+
+  const props = defineProps<{
+    flowRunId: string,
+  }>()
+
+  const showModal = defineModel<boolean>('showModal')
+  const api = useWorkspaceApi()
+  const parameters = ref<SchemaValuesV2>({})
```

Comment:
Just double checking - downstream schema inputs will populate this with defaults, right?

Reply:
Correct, the component itself will automatically fill in the default values. That way you don't have to worry about it at implementation.